bindata/v3.11.0/kube-controller-manager/defaultconfig.yaml

```diff
@@ -13,6 +13,8 @@ extendedArguments:
   - "10.3.0.0/16"
   use-service-account-credentials:
   - "true"
+  flex-volume-plugin-dir:
```

Comment:
Please add a comment indicating that storage owns this and you specifically

Reply:
ack